stub-runner/stub-runner/src/main/groovy/com/ofg/stub/registry/StubRegistry.groovy

```diff
@@ -54,7 +55,18 @@ class StubRegistry {
                 .uriSpec(URI_SPEC)
                 .address('localhost')
                 .port(stubServer.port)
-                .name(stubServer.projectMetadata.projectRelativePath)
+                .name(serviceName(stubServer))
                 .build()
     }
+
+    private static String serviceName(StubServer stubServer) {
+        ProjectMetadata metadata = stubServer.projectMetadata
+        String projectRelativePath = metadata.projectRelativePath
```

Comment:
No, since the method is called on collection of `StubServer`s. And `metadata` are required to create `StubServer` so I assume it can not be `null`.